config.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from functools import lru_cache
+from typing import Union, Optional
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from pydantic import SecretStr, PostgresDsn, DirectoryPath, Field, validator
+
+logger = logging.getLogger(__name__)
+
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_file="/home/abdullah/Documents/hdd/projects/ansari/ansari-backend/.env", env_file_encoding="utf-8", case_sensitive=True)
+    
+    DATABASE_URL: PostgresDsn = Field(default="postgresql://mwk@localhost:5432/mwk")
+    MAX_THREAD_NAME_LENGTH: int = Field(default=100)
+
+    SECRET_KEY: SecretStr = Field(default="secret")
+    ALGORITHM: str = Field(default="HS256")
+    ENCODING: str = Field(default="utf-8")
+    ACCESS_TOKEN_EXPIRY_HOURS: int = Field(default=2)
```

Comment:
Thank you so much for including time units in variable names!

Reply:
Thanks for your kind comment 😊